### PR TITLE
Fix konami_code_fsm startup after #2375

### PR DESCRIPTION
**Pull request by skorobkov**
Konami throw error on starup of `konami_code_fsm`
> Aug  5 04:24:54 t13-devapps 2600hz[16323]: |YTEyNzkxNTE5MzViZjE1MzZiYzkxMTExOWVmYzEwYzg.|konami_code_fsm:308 (<0.14502.0>) using custom binding digit '*'
Aug  5 04:24:54 t13-devapps 2600hz[16323]: |YTEyNzkxNTE5MzViZjE1MzZiYzkxMTExOWVmYzEwYzg.|konami_listener:83 (<0.14502.0>) failed to run FSM: exit: process_was_not_started_by_proc_lib
Aug  5 04:24:54 t13-devapps 2600hz[16323]: |YTEyNzkxNTE5MzViZjE1MzZiYzkxMTExOWVmYzEwYzg.|kz_util:169 (<0.14502.0>) stacktrace:
Aug  5 04:24:54 t13-devapps 2600hz[16323]: |YTEyNzkxNTE5MzViZjE1MzZiYzkxMTExOWVmYzEwYzg.|kz_util:176 (<0.14502.0>) st: gen_fsm:get_parent/0 at (350)
Aug  5 04:24:54 t13-devapps 2600hz[16323]: |YTEyNzkxNTE5MzViZjE1MzZiYzkxMTExOWVmYzEwYzg.|kz_util:176 (<0.14502.0>) st: gen_fsm:enter_loop/6 at (309)
Aug  5 04:24:54 t13-devapps 2600hz[16323]: |YTEyNzkxNTE5MzViZjE1MzZiYzkxMTExOWVmYzEwYzg.|kz_util:176 (<0.14502.0>) st: konami_listener:handle_metaflow/2 at (73)

This hapens after 66924e3687394f42be0459f2a4a2cc0b15505fc9 (#2375) where `gen_listener` start using `kz_util:spawn/2` instead of `proc_lib:spawn/3` for `gen_listener:distribute_event/4`

### CODE REVIEW

File: applications/konami/src/konami_code_fsm.erl [hunk 1]
```diff
@@ -83,38 +83,9 @@
 %%% API
 %%%===================================================================
 
--spec start_fsm(kapps_call:call(), kz_json:object()) -> any().
+-spec start_fsm(apps_call:call(), kz_json:object()) -> any().
```
Comment (jamesaimonetti) [inline applications/konami/src/konami_code_fsm.erl]: I don't think you meant to change this type? It should remain `kapps_call:call()`

Comment (skorobkov) [inline applications/konami/src/konami_code_fsm.erl]: Yes. Its typo.

**Comment by jamesaimonetti** [comment]
@skorobkov thanks, I've manually merged this in (haven't figured out how to get Github to auto-detect it).

**Comment by fenollp** [comment]
@lazedo @jamesaimonetti What does the change from proc_lib:spawn to erlang:spawn imply in #2375 ?

**Pull request closed without merging**